python/hail/docs/annotationdb.rst

```diff
@@ -0,0 +1,234 @@
+.. _sec-annotationdb:
+
+===================
+Annotation Database
+===================
+
+This database contains a curated collection of variant annotations in Hail-friendly format, for use in Hail analysis pipelines. 
+
+.. _here: https://console.cloud.google.com/storage/browser/annotationdb/?project=broad-ctsa
+
+Currently, the :code:`annotate_variants_db()` VDS method associated with this database works only if you are running Hail on the 
```

Comment:
make this the link rather than putting a "method" link after. It should also be relative: `:py:meth:`~.VariantDataset.annotate_variants_db` should do it, I think